File: q1/quest1.py
sum_val=0
max_val=1000

for i in range(1,max_val+1):
	if (i % 3 == 0) or (i % 5 == 0):
		sum_val = sum_val + i

print("the sum of all integer multiples of 3 or 5 less than " + str(max_val) + " is " + str(sum_val))

File: q1/quest1.cpp
#include <stdio.h>

int main() {
	int p1 = 3;
	int p2 = 5;
	int max_val = 1000;
	int sum = 0;

	for (int i = 0; i < max_val; i++)
		if (i % p1 == 0 || i % p2 == 0) {
			sum += i;
		}

	printf("the sum of all integer multiples of %d or %d less than %d is %d\n", p1, p2, max_val, sum);
}